scripts/west_commands/sign.py

```diff
@@ -9,12 +9,14 @@
 import pickle
 import platform
 import shutil
+import shlex
 import subprocess
 import sys
 
 from west import log
 from west import manifest
 from west.util import quote_sh_list
+from west.configuration import config as west_config
```

Comment:
This is deprecated; please use `WestCommand.config` instead. https://docs.zephyrproject.org/latest/develop/west/west-apis.html#westcommand